src/cargo/core/workspace.rs

```diff
@@ -103,6 +103,9 @@ pub struct Workspace<'gctx> {
     // file. This is set for `cargo install` without `--locked`.
     ignore_lock: bool,
 
+    // Requested path of the lockfile (i.e. passed as the cli flag)
+    requested_lockfile_path: Option<PathBuf>,
```

Comment:
Use doc comment is preferrable.

(Just notice some fields here should switch to doc comemnt as well)

```suggestion
    /// Requested path of the lockfile (i.e. passed as the cli flag)
    requested_lockfile_path: Option<PathBuf>,
```

Reply:
> (Just notice some fields here should switch to doc comemnt as well)

Addressed it in #14397 before it is forgotten